src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/ReleaseNotes.txt

```diff
@@ -1 +1 @@
-﻿December 2017: v2.6.0 Preview Release
\ No newline at end of file
+﻿January 2018: v2.6.0 Official Release
```

Comment:
Do we want to mention the minimum supported VS version (15.5) in the release notes?

Reply:
That's probably a good idea.